src/net.h

```diff
@@ -691,12 +691,18 @@ class CNode
     uint256 hashContinue;
     std::atomic<int> nStartingHeight{-1};
 
-    // flood relay
-    std::vector<CAddress> vAddrToSend;
-    CRollingBloomFilter addrKnown;
-    bool fGetAddr{false};
-    int64_t nNextAddrSend GUARDED_BY(cs_sendProcessing){0};
-    int64_t nNextLocalAddrSend GUARDED_BY(cs_sendProcessing){0};
+    struct AddrRelay {
+        CCriticalSection cs_addrsend;
+        std::vector<CAddress> vAddrToSend GUARDED_BY(cs_addrsend);
+        CRollingBloomFilter addrKnown GUARDED_BY(cs_addrsend);
```

Comment:
https://github.com/bitcoin/bitcoin/pull/15759/commits/c36939ca2ea636dce8047065d88c2e70bd418405

Move initialization here and drop constructor:
```cpp
CRollingBloomFilter addrKnown GUARDED_BY(cs_addrsend){5000, 0.001};
```

Reply:
Thanks, fixed.